turbinia/config/turbinia_config_tmpl.py

```diff
@@ -51,6 +51,13 @@
 # Path to a lock file used for the worker tasks.
 LOCK_FILE = '%s/turbinia-worker.lock' % OUTPUT_DIR
 
+# Path to a resource state file used for tracking shared Evidence types.
+RESOURCE_FILE = '%s/turbinia-state.json' % OUTPUT_DIR
```

Comment:
This should be a shared path between workers running on the same machine. So OUTPUT_DIR should be eg mapped into a containers from the host, else each worker will have their own resource lock file. 
We should document this in one of the installation files and/or add a more descriptive comment here.

Reply:
Same for RESOURCE_FILE_LOCK

Reply:
Thanks, updated install.md and config file to better explain resource state file!